lib/tests/testscripts/scripts/reporting/vte-collect-data.xml

```diff
@@ -0,0 +1,333 @@
+                <contentType value="json"/>
+                <params value="/MitreTestScript-library-vte-icu-FHIR"/>
+                <sourceId value="library-vte-icu-FHIR"/>
+            </operation>
+        </action>
+        <action>
+            <assert>
+                <description value="Confirm that the returned HTTP status is 200(OK) or 201(Created)."/>
+                <operator value="in"/>
+                <responseCode value="200,201"/>
+            </assert>
+        </action>
+        <action>
+            <operation>
+                <type>
+                    <system value="http://hl7.org/fhir/testscript-operation-codes"/>
+                    <code value="update"/>
+                </type>
+                <resource value="Library"/>
+                <description value="Update or create a library on the server"/>
+                <accept value="json"/>
+                <contentType value="json"/>
+                <params value="/MitreTestScript-library-vte-1-FHIR"/>
+                <sourceId value="library-vte-1-FHIR"/>
+            </operation>
+        </action>
+        <action>
+            <assert>
+                <description value="Confirm that the returned HTTP status is 200(OK) or 201(Created)."/>
+                <operator value="in"/>
+                <responseCode value="200,201"/>
+            </assert>
+        </action>
+        <action>
+            <operation>
+                <type>
+                    <system value="http://hl7.org/fhir/testscript-operation-codes"/>
+                    <code value="update"/>
+                </type>
+                <resource value="Measure"/>
+                <description value="Update or create a measure on the server"/>
+                <accept value="json"/>
+                <contentType value="json"/>
+                <params value="/MitreTestScript-measure-vte-1-FHIR"/>
+                <sourceId value="measure-vte-1-FHIR"/>
+            </operation>
+        </action>
+        <action>
+            <assert>
+                <description value="Confirm that the returned HTTP status is 200(OK) or 201(Created)."/>
+                <operator value="in"/>
+                <responseCode value="200,201"/>
+            </assert>
+        </action>
+    </setup>
+
+    <test id="TestCollectData">
+        <name value="Test Collect Data"/>
+        <description value="Tests the $collect-data operation for the VTE measure"/>
+        <action>
+            <operation>
+                <type>
+                    <system value="http://hl7.org/fhir/testscript-operation-codes"/>
+                    <code value="collect-data"/>
+                </type>
+                <resource value="Measure"/>
+                <description value="Collect data for the measure"/>
+                <accept value="json"/>
+                <contentType value="json"/>
+                <params value="/${createMeasureId}/$collect-data?periodStart=${periodStart}&amp;periodEnd=${periodEnd}"/>
```

Comment:
`${createMeasureId}` should be replaced with `MitreTestScript-measure-vte-1-FHIR`

Reply:
Good catch